src/utils.rs

```diff
@@ -419,6 +421,23 @@ impl Style {
     }
 }
 
+impl Style {
+    pub fn get_fg(&self) -> Option<Color> {
```

Comment:
Why are these APIs added? Let's not add these unless there is a reason (maybe follow up PR), and if they are added, they need docs.